### ISSUE TITLE
When I have no data in a column, and I sort the column, I am logged out

### ISSUE THREAD

**Issue by csprayregen** [pieforproviders/pieforproviders#962]
## 🤬 What's wrong?
If I click the arrows to sort a column and there is no data in that column, I am logged out of my account and get an error message (screenshot below). This only seems to happen when there is no data. 

## 🏓 Implementation Details

> **NOTE TO ENGINEERS**
> 
> Please ensure that you are running Google Lighthouse and/or WebAIM (Wave) on any UI elements in your pull request to check for accessibility issues.
> 
> If it is out of scope to address those accessibility issues in your pull request (because the issue is in a parent component and needs refactoring to work, for example), **you are responsible for noting those issues in your pull request and writing bug tickets to address the issues - make sure you link the new tickets in your pull request**

**Comment by katelovescode** [comment]
@csprayregen @cjhaddad can we verify if this is still happening?

**Comment by csprayregen** [comment]
@katelovescode yes but probably need you to generate some staging data for me that matches this use case? Unless there's a better way?

**Comment by katelovescode** [comment]
Ya we can do this after the dashboard QA is done
